build/ci/.azure-devops-android-tests.yml

```
@@ -142,6 +142,8 @@ jobs:
           FAILBUILD_ON_FAILURE: true
           ALLOW_RERUN: true
           UITEST_TEST_TIMEOUT: '270000'
+          ANDROID_VERSION: '9.0'
+          JobDiplayName: 'Android $(ANDROID_VERSION) api $(ANDROID_SIMULATOR_APILEVEL) $(TARGETPLATFORM_NAME) UI Tests Group $(UNO_UITEST_BUCKET_ID)'
```

I wouldn't introduce a new variable here just to use in display name

Why not something like this?

```suggestion
          JobDiplayName: 'Android 9.0 api $(ANDROID_SIMULATOR_APILEVEL) $(TARGETPLATFORM_NAME) UI Tests Group $(UNO_UITEST_BUCKET_ID)'
```